### PR TITLE
Add action history button link

### PR DESCRIPTION
**Pull request by laviro**
Rename `DocsButton` to `ToolbarButtons` and extend it with the history button link.

### CODE REVIEW

File: webpack/ForemanInventoryUpload/ForemanInventoryConstants.js [hunk 1]
```diff
@@ -3,3 +3,5 @@ import { translate as __ } from 'foremanReact/common/I18n';
 export const INVENTORY_PAGE_TITLE = __('Red Hat Inventory Uploads');
 
 export const DOCS_BUTTON_TEXT = __(' Documentation');
+
+export const TASKS_HISTORY_BUTTON_TEXT = __(' Tasks history');
```
Comment (ShimShtein) [inline webpack/ForemanInventoryUpload/ForemanInventoryConstants.js:7]: I think `Actions history` is more understandable for the user. Tasks are implementation details here.

Comment (laviro) [reply webpack/ForemanInventoryUpload/ForemanInventoryConstants.js:7]: no problem.. :+1:  I thought about `tasks` from the `tasks page`

File: webpack/ForemanInventoryUpload/Components/PageHeader/components/DocsButton/DocsButton.js [hunk 2]
```diff
@@ -2,11 +2,9 @@ import React from 'react';
 import { Button, Icon } from 'patternfly-react';
 import { DOCS_BUTTON_TEXT } from '../../../../ForemanInventoryConstants';
 import { getInventoryDocsUrl } from '../../../../ForemanInventoryHelpers';
-import './docsButton.scss';
 
-const DocsButton = () => (
+const ToolbarButtons = () => (
```
Comment (ShimShtein) [inline webpack/ForemanInventoryUpload/Components/PageHeader/components/DocsButton/DocsButton.js:6]: It exports only the `DocsButton`, isn't it?

Comment (laviro) [reply webpack/ForemanInventoryUpload/Components/PageHeader/components/DocsButton/DocsButton.js]: yes, it was a leftover from before splitting the component, fixed!

**Comment by ShimShtein** [comment]
Merged, thanks @laviro!

**Pull request merged**